### ISSUE TITLE
apt.deb(src="http://url/here") breaks in facts/deb.py

### ISSUE THREAD

**Issue by drewp** [Fizzadar/pyinfra#702]
My deploy says this:
```
    apt.deb(src='http://http.us.debian.org/debian/pool/main/n/netplan.io/netplan.io_0.95-2_armhf.deb')
```

I inserted this print in facts/deb.py:
```
    # We have deb info! Check against installed packages
    print(f'{src=} {info=}')
    if info:
```
and I get this output:
```
[garage] >>> sh -c '! test -e /tmp/pyinfra-7ed098bf43cef74d8ab8ea095e4a95c92605c61c || ( stat -c '"'"'user=%U group=%G mode=%A atime=%X mtime=%Y ctime=%Z size=%s %N'"'"' /tmp/pyinfra-7ed098bf43cef74d8ab8ea095e4a95c92605c61c 2> /dev/null || stat -f '"'"'user=%Su group=%Sg mode=%Sp atime=%a mtime=%m ctime=%c size=%z %N%SY'"'"' /tmp/pyinfra-7ed098bf43cef74d8ab8ea095e4a95c92605c61c )'
    Loaded fact file (path=/tmp/pyinfra-7ed098bf43cef74d8ab8ea095e4a95c92605c61c)
    [garage] noop: file /tmp/pyinfra-7ed098bf43cef74d8ab8ea095e4a95c92605c61c has already been downloaded
[garage] >>> sh -c '! command -v dpkg >/dev/null || ! test -e /tmp/pyinfra-7ed098bf43cef74d8ab8ea095e4a95c92605c61c && (dpkg -s /tmp/pyinfra-7ed098bf43cef74d8ab8ea095e4a95c92605c61c 2>/dev/null || true) || dpkg -I /tmp/pyinfra-7ed098bf43cef74d8ab8ea095e4a95c92605c61c'
    Loaded fact deb_package (name=/tmp/pyinfra-7ed098bf43cef74d8ab8ea095e4a95c92605c61c)
[garage] >>> sh -c '! command -v dpkg >/dev/null || dpkg -l'
    Loaded fact deb_packages ()
src='/tmp/pyinfra-7ed098bf43cef74d8ab8ea095e4a95c92605c61c' info={'version': '0.95-2'}
--> An unexpected exception occurred in: net.py:

  File "/stor6/my/proj/infra/env/lib/python3.8/site-packages/pyinfra_cli/util.py", line 80, in exec_file
    exec(PYTHON_CODES[filename], data)
  File "net.py", line 22, in <module>
    apt.deb(src='http://http.us.debian.org/debian/pool/main/n/netplan.io/netplan.io_0.95-2_armhf.deb')
  File "/stor6/my/proj/infra/env/lib/python3.8/site-packages/pyinfra/api/operation.py", line 365, in decorated_func
    commands = unroll_generators(func(*actual_args, **actual_kwargs))
  File "/stor6/my/proj/infra/env/lib/python3.8/site-packages/pyinfra/api/util.py", line 192, in unroll_generators
    for item in generator:
  File "/stor6/my/proj/infra/env/lib/python3.8/site-packages/pyinfra/operations/apt.py", line 247, in deb
    info['name'] in current_packages
KeyError: 'name'
```

--> Support information:

    If you are having issues with pyinfra or wish to make feature requests, please
    check out the GitHub issues at https://github.com/Fizzadar/pyinfra/issues .
    When adding an issue, be sure to include the following:

    System: Linux
      Platform: Linux-5.4.0-89-generic-x86_64-with-glibc2.29
      Release: 5.4.0-89-generic
      Machine: x86_64
    pyinfra: v1.4.18
    Executable: env/bin/pyinfra
    Python: 3.8.10 (CPython, GCC 9.3.0)

**Comment by drewp** [comment]
Here's a silly workaround:

```
@task
def net(ctx):
    # workaround for https://github.com/Fizzadar/pyinfra/issues/702
    ctx.run(cmd + '-vv inventory.py exec -- rm -f /tmp/pyinfra-7ed098bf43cef74d8ab8ea095e4a95c92605c61c', pty=True)

    ctx.run(cmd + '-vv inventory.py net.py', pty=True)
```

**Comment by Fizzadar** [comment]
Thank you for reporting this @drewp; I've just pushed up a fix in https://github.com/Fizzadar/pyinfra/commit/0abfd55fe0e33d570f6cf3251fcba315967b4dde pending release shortly.

**Comment by Fizzadar** [comment]
Now released in v1.4.19.
